dockstore-cli-integration-testing/generated/src/main/resources/pom.xml

```diff
@@ -49,13 +49,19 @@
     <dependency>
       <groupId>io.dockstore</groupId>
       <artifactId>dockstore-common</artifactId>
-      <version>1.17.0-alpha.0</version>
+      <version>1.17.0-alpha.2</version>
       <scope>compile</scope>
+      <exclusions>
```

Comment:
Note: this library is used by the webservice when it uses a library (org.kohsuke github-api) It can safely be excluded by the CLI (confirmed by these tests)
I do not know why the webservice build has not died for the same reason but it it does, you can exclude the key in the keylist file as we do with the jackson libraries. 

We could also try upgrading as noted by 
https://github.com/auth0/java-jwt